src/main/java/edu/harvard/iq/dataverse/datavariable/DataVariable.java

```diff
@@ -414,13 +426,21 @@ public void setUnf(String unf) {
     public Collection<VariableCategory> getCategories() {
         return this.categories;
     }
+
+    public boolean isLabled () {
```

Comment:
I'm not sure if I should remove `isLabled` the theory behind it is that it represents the fact that the SPSS and Stata way of looking at these is different from that of R, but on the other hand, it adds code that currently doesn't really have a consumer (as UNF ignores it currently, and we don't ask if something is categorical anywhere else)